.azure-pipelines/ci-build.yml

```
@@ -21,248 +18,252 @@ variables:
+            condition: and(contains(variables['build.sourceBranch'], 'refs/heads/main'), succeeded())
+            inputs:
+              SourceFolder: '$(Build.SourcesDirectory)'
+              Contents: 'scripts\**'
+              TargetFolder: '$(Build.ArtifactStagingDirectory)/Nugets'
 
-      - task: PublishPipelineArtifact@1
-        displayName: 'Upload Artifact: Nugets'
-        inputs:
-          artifactName: Nugets
-          targetPath: $(Build.ArtifactStagingDirectory)/Nugets
+          - task: PublishPipelineArtifact@1
+            displayName: 'Upload Artifact: Nugets'
+            inputs:
+              artifactName: Nugets
+              targetPath: $(Build.ArtifactStagingDirectory)/Nugets
 
-- stage: deploy
-  condition: and(contains(variables['build.sourceBranch'], 'refs/heads/main'), succeeded())
-  dependsOn: build
-  jobs:
-    - deployment: deploy_cli_commons
-      environment: nuget-org
-      strategy:
-        runOnce:
-          deploy:
-            pool:
-              vmImage: ubuntu-latest
-            steps:
-            - task: DownloadPipelineArtifact@2
-              displayName: Download nupkg from artifacts
-              inputs:
-                artifact: Nugets
-                source: current
-            - task: PowerShell@2
-              displayName: 'Extract release information to pipeline'
-              inputs:
-                targetType: 'filePath'
-                filePath: $(Pipeline.Workspace)\Nugets\scripts\GetNugetPackageVersion.ps1
-                pwsh: true
-                arguments: '-packageDirPath "$(Pipeline.Workspace)/Nugets/"'
-            - task: NuGetCommand@2
-              displayName: 'NuGet push'
-              inputs:
-                command: push
-                packagesToPush: '$(Pipeline.Workspace)/Nugets/Microsoft.Kiota.Cli.Commons.*.nupkg'
-                nuGetFeedType: external
-                publishFeedCredentials: 'Kiota Nuget Connection'
+    - stage: deploy
+      condition: and(contains(variables['build.sourceBranch'], 'refs/heads/main'), succeeded())
+      dependsOn: build
+      jobs:
+        - deployment: deploy_cli_commons
+          environment: nuget-org
+          strategy:
+            runOnce:
+              deploy:
+                pool:
+                  vmImage: ubuntu-latest
+                steps:
+                - task: DownloadPipelineArtifact@2
+                  displayName: Download nupkg from artifacts
+                  inputs:
+                    artifact: Nugets
+                    source: current
+                - task: PowerShell@2
+                  displayName: 'Extract release information to pipeline'
+                  inputs:
+                    targetType: 'filePath'
+                    filePath: $(Pipeline.Workspace)\Nugets\scripts\GetNugetPackageVersion.ps1
+                    pwsh: true
+                    arguments: '-packageDirPath "$(Pipeline.Workspace)/Nugets/"'
+                - task: NuGetCommand@2

```

This has to be as below for the pipeline to run.
```suggestion
                - task: 1ES.PublishNuget@1
```